includes/class-wc-gateway-ppec-cart-handler.php

```diff
@@ -458,17 +458,37 @@ public function enqueue_scripts() {
 			);
 
 		} elseif ( 'yes' === $settings->use_spb ) {
-			wp_register_script( 'paypal-checkout-js', 'https://www.paypalobjects.com/api/checkout.js', array(), null, true );
-			wp_register_script( 'wc-gateway-ppec-smart-payment-buttons', wc_gateway_ppec()->plugin_url . 'assets/js/wc-gateway-ppec-smart-payment-buttons.js', array( 'jquery', 'paypal-checkout-js' ), wc_gateway_ppec()->version, true );
+			// TODO: Add client ID.
+			$client_id = 'sandbox' === $settings->get_environment() ? 'sb' : '<client_id>';
+			// TODO: Add commit true/false.
```

Comment:
What are these? Include slightly longer explanations. Extra credit for URLs to documentation.